letsencrypt-auto-source/pieces/bootstrappers/rpm_common.sh

```diff
@@ -25,9 +25,9 @@ BootstrapRpmCommon() {
   fi
 
   if ! $SUDO $tool list *virtualenv >/dev/null 2>&1; then
-    echo "To use Certbot, packages from the EPEL repository need to be installed."
+    say "To use Certbot, packages from the EPEL repository need to be installed."
```

Comment:
We could also wrap `sudo` the way we do for `su` to echo the command we're about to run, but many users only have to type in their `sudo` password the first time meaning we'll be printing a lot of unnecessary output. Ideally we find a way to only print the message if we know we're going to prompt for a password. We could do this by capturing the output, but I want to make sure whatever we do isn't too complex and is robust on the wide variety of systems `certbot-auto` is run on.